ctrd/container.go

```diff
@@ -385,7 +385,10 @@ func (c *Client) destroyContainer(ctx context.Context, id string, timeout int64)
 		}
 		msg = waitExit()
 	}
-	if err := msg.RawError(); err != nil && errtypes.IsTimeout(err) {
+
+	// ignore the error is stop time out
+	// TODO: how to design the stop error is time out?
```

Comment:
Should we add a hook plugin to deal with the time-out case? or the containerd-shim may alive but pouchd think it was stopped

Reply:
@Ace-Tang your suggestion maybe useful, but it just a little ugly in codes.